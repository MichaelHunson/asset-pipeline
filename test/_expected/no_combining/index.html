<link href='/coffee.css' media='all' rel='stylesheet' type='text/css'><link href='/test.css' media='all' rel='stylesheet' type='text/css'><link href='/sass-test.css' media='all' rel='stylesheet' type='text/css'>
<script src='/foo.js'></script><script src='/blah.js'></script>
