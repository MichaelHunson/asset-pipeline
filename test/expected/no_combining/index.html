<script src='/foo.js'></script><script src='/blah.js'></script>
<link href='/test.css' media='all' rel='stylesheet' type='text/css'><link href='/css-test.scss' media='all' rel='stylesheet' type='text/css'>
